lib/connection.js

```diff
@@ -241,33 +237,36 @@ class Connection extends EventEmitter {
   // Send a pong message
   //
   pong(messageId) {
+    if (this._closed) {
```

Comment:
Given that it's a public method, I'd say that the caller shouldn't invoke `pong()` if the connection is closed. Otherwise, this check makes `pong()` inconsistent with other methods of a connection in a way that one may call it on a closed connection without any errors, which doesn't sound right.

Reply:
@aqrln mark `pong` as a private method, there is no reason for it to be public.

Reply:
@nechaido sounds good to me, but the same goes for `callback`, then. (Though it's kinda out of scope of this PR.)